.github/workflows/main.yaml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  lint-report:
+    uses: canonical/vault-k8s-operator/.github/workflows/lint-report.yaml@main
+
+  static-analysis:
+    uses: canonical/vault-k8s-operator/.github/workflows/static-analysis.yaml@main
+
+  unit-tests-with-coverage:
+    uses: canonical/vault-k8s-operator/.github/workflows/unit-test.yaml@main
+
+  integration-test:
+    uses: canonical/vault-k8s-operator/.github/workflows/integration-test.yaml@main
+    with:
+      charm-file-name: "vault-k8s_ubuntu-22.04-amd64.charm"
```

Comment:
`charm-file-name` is not used in the `integration-test` workflow. We probably want to update `integration-test.yaml` to have something similar to the sdcore-github-workflows' [file](https://github.com/canonical/sdcore-github-workflows/blob/main/.github/workflows/integration-test.yaml)

Reply:
Done